packages/web/src/scss/components/UNSTABLE_ProductLogo/_UNSTABLE_ProductLogo.scss

```
@@ -0,0 +1,18 @@
+@use 'sass:map';
+@use 'theme' as theme;
+@use '../../tools/breakpoint';
+
+.UNSTABLE_ProductLogo {
+    width: fit-content;
+}
+
+.UNSTABLE_ProductLogo > img,
+svg {
+    width: auto;
+    height: 44px;
```

Should be part of the `_theme.scss`.
```suggestion
    height: theme.$logo-height;
```